WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -386,27 +378,27 @@ private extension ZendeskUtils {
 
         guard let userEmail = ZendeskUtils.sharedInstance.userEmail else {
             DDLogInfo("No user email to create Zendesk identity with.")
-            ZDKConfig.instance().userIdentity = nil
+            let identity = Identity.createAnonymous()
+            Zendesk.instance?.setIdentity(identity)
             completion(false)
             return
         }
 
-        let zendeskIdentity = ZDKAnonymousIdentity()
-        zendeskIdentity.email = userEmail
-        zendeskIdentity.name = ZendeskUtils.sharedInstance.userName
-        ZDKConfig.instance().userIdentity = zendeskIdentity
-        DDLogDebug("Zendesk identity created with email '\(zendeskIdentity.email ?? "")' and name '\(zendeskIdentity.name ?? "")'.")
+        let zendeskIdentity = Identity.createAnonymous(name: ZendeskUtils.sharedInstance.userName, email: userEmail)
+        Zendesk.instance?.setIdentity(zendeskIdentity)
+        DDLogDebug("Zendesk identity created with email '\(userEmail)' and name '\(ZendeskUtils.sharedInstance.userName ?? "")'.")
         registerDeviceIfNeeded()
         completion(true)
     }
 
     static func registerDeviceIfNeeded() {
 
-        guard let deviceID = ZendeskUtils.sharedInstance.deviceID else {
+        guard let deviceID = ZendeskUtils.sharedInstance.deviceID,
+        let zendeskInstance = Zendesk.instance else {
             return
         }
 
-        ZDKConfig.instance().enablePush(withDeviceID: deviceID) { pushResponse, error in
+        ZDKPushProvider(zendesk: zendeskInstance).register(deviceIdentifier: deviceID, locale: appLanguage) { (pushResponse, error) in
```

Comment:
Regarding the addition of parenthesis to the capture list, I see a mix of conventions in this file:

- Some, like `static ZendeskUtils.getUserInformationFrom(blog:completion:)` leave them off.
- Still others, like `static ZendeskUtils.promptUserForInformation(withName:completion:)` appear to do both.

I'm not familiar with the team convention; would it be possible to make this file consistent throughout?

Reply:
Hrm. I don't know that there's a team convention so to speak. I've seen both methods used throughout our project. I tend to only use parenthesis if there are multiple parameters, as in this case.